dynamiq/nodes/retrievers/weaviate.py

```diff
@@ -8,9 +8,10 @@
 from dynamiq.nodes.node import NodeGroup, VectorStoreNode, ensure_config
 from dynamiq.runnables import RunnableConfig
 from dynamiq.storages.vector import WeaviateVectorStore
+from dynamiq.storages.vector.base import BaseVectorStoreParams
 
 
-class WeaviateDocumentRetriever(VectorStoreNode):
+class WeaviateDocumentRetriever(VectorStoreNode, BaseVectorStoreParams):
```

Comment:
parent `VectorStoreNode` already inherited from `BaseVectorStoreParams`, so all this params exists autimatically.